### PR TITLE
chore: fix spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/prisma/actions/runs/3178811637#summary-8690446420

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/prisma/commit/0eccc41e21b71bef21cf0dd0416c0526c9db603f
 
Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: packages/generator-helper/src/__tests__/generatorHandler.test.ts [hunk 1]
```diff
@@ -95,7 +95,7 @@ describe('generatorHandler', () => {
             "SomeForbiddenModel",
           ],
         },
-        "prettyName": "This is a pretty pretty name",
+        "prettyName": "This is a pretty name",
```
Comment (jsoref) [inline packages/generator-helper/src/__tests__/generatorHandler.test.ts:98]: It's possible this is trying to be cute...

File: packages/internals/src/__tests__/getGitHubIssueUrl.test.ts [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import stripAnsi from 'strip-ansi'
 
-import { getGithubIssueUrl } from '../utils/getGithubIssueUrl'
+import { getGitHubIssueUrl } from '../utils/getGitHubIssueUrl'
```
Comment (jsoref) [inline packages/internals/src/__tests__/getGitHubIssueUrl.test.ts:3]: I believe I've properly renamed everything for this change...

File: packages/internals/src/utils/isCi.ts [hunk 3]
```diff
@@ -7,7 +7,7 @@ export const isCi = (): boolean => {
   // This list is probably not exhaustive though `process.env.CI` should be enough
   //  but since we were using this utility in the past, we want to keep the same behavior
   return !!(
-    env.CI || // Travis CI, CircleCI, Cirrus CI, Gitlab CI, Appveyor, CodeShip, dsari
+    env.CI || // Travis CI, CircleCI, Cirrus CI, GitLab CI, Appveyor, CodeShip, dsari
```
Comment (jsoref) [inline packages/internals/src/utils/isCi.ts:10]: Brand

File: packages/migrate/src/__tests__/fixtures/existing-db-1-draft-1-change/prisma/schema.prisma [hunk 4]
```diff
@@ -15,5 +15,5 @@ model Blog {
 
 model Cat {
   id    Int @id
-  lifes Int
+  lives Int
```
Comment (jsoref) [inline packages/migrate/src/__tests__/fixtures/existing-db-1-draft-1-change/prisma/schema.prisma:18]: I'm assuming this is purely test data and doesn't have any corresponding anything...

**Review by jsoref — COMMENTED** [review]
Many changes were suggested by Google Sheets. All fault mine.

File: .github/workflows/benchmark.yml [hunk 5]
```diff
@@ -74,7 +74,7 @@ jobs:
           name: Benchmark.js Benchmark
           tool: 'benchmarkjs'
           output-file-path: output.txt
-          # Use personal access token instead of GITHUB_TOKEN due to https://github.community/t5/GitHub-Actions/Github-action-not-triggering-gh-pages-upon-push/td-p/26869/highlight/false
+          # Use personal access token instead of GITHUB_TOKEN due to https://github.community/t5/GitHub-Actions/GitHub-action-not-triggering-gh-pages-upon-push/td-p/26869/highlight/false
```
Comment (Jolg42) [inline .github/workflows/benchmark.yml:77]:
```suggestion
          # Note: must use personal access token
```

File: packages/client/scripts/postinstall.js [hunk 6]
```diff
@@ -282,7 +282,7 @@ function makeDir(input) {
  * an error while attempting to get this value then the string constant
  * 'ERROR_WHILE_FINDING_POSTINSTALL_TRIGGER' is returned.
  * This information is just necessary for telemetry.
- * This get's passed in to Generate, which then automatically get's propagated to telemetry.
+ * This get's passed into Generate, which then automatically get's propagated to telemetry.
```
Comment (Jolg42) [inline packages/client/scripts/postinstall.js:285]:
```suggestion
 * This is passed to `prisma generate` as a string like `--postinstall value`.
```

We can probably change to this

File: packages/client/src/runtime/getPrismaClient.ts [hunk 7]
```diff
@@ -625,7 +625,7 @@ export function getPrismaClient(config: GetPrismaClientConfig) {
         queryString = query
         parameters = {
           values: serializeRawParameters(values || []),
-          __prismaRawParamaters__: true,
+          __prismaRawParameters__: true,
```
[the remark this answers is not in the document]
Comment (Jolg42) [reply packages/client/src/runtime/getPrismaClient.ts:628]: 🙈 Yes, also quick check to confirm: https://cs.github.com/?scopeName=All+repos&scope=&q=__prismaRawParamaters__#

File: packages/client/tests/functional/issues/4004/tests.ts [hunk 8]
```diff
@@ -74,7 +74,7 @@ testMatrix.setupTestSuite(({ provider }) => {
       },
     })
 
-    // Read all the join tabels
+    // Read all the join tables
```
[the remark this answers is not in the document]
Comment (Jolg42) [reply packages/client/tests/functional/issues/4004/tests.ts:77]: tables looks correct

File: packages/fetch-engine/src/downloadZip.ts [hunk 9]
```diff
@@ -102,7 +102,7 @@ export async function downloadZip(
           }
 
           if (hash !== sha256) {
-            throw new Error(`sha256 of ${url} (uzipped) should be ${sha256} but is ${hash}`)
+            throw new Error(`sha256 of ${url} (unzipped) should be ${sha256} but is ${hash}`)
```
[the remark this answers is not in the document]
Comment (Jolg42) [reply packages/fetch-engine/src/downloadZip.ts:105]: Yes!

File: .github/workflows/test.yml [hunk 10]
```diff
@@ -1095,7 +1095,7 @@ jobs:
           name: engines-${{ matrix.os }}
 
       - name: Upload test results to BuildPulse for flaky test detection
-        # Only run this step for branches where where we have access to secrets.
```
[the remark this answers is not in the document]
Comment (Jolg42) [reply .github/workflows/test.yml:1098]: Looks good!

File: .github/workflows/benchmark.yml (hunk 5 above)
[the remark this answers is not in the document]
Comment (Jolg42) [reply .github/workflows/benchmark.yml:77]: Suggestion here https://github.com/prisma/prisma/pull/15645#discussion_r986539608

File: packages/client/src/__tests__/validatePrismaClientOptions.test.ts [hunk 11]
```diff
@@ -45,12 +45,12 @@ describe('invalid options', () => {
...
       validatePrismaClientOptions(
         {
-          errorrFormat: 'minimal',
+          errorsFormat: 'minimal',
         } as any,
         ['db'],
       ),
     ).toThrowErrorMatchingInlineSnapshot(`
-      Unknown property errorrFormat provided to PrismaClient constructor. Did you mean "errorFormat"?
+      Unknown property errorsFormat provided to PrismaClient constructor. Did you mean "errorFormat"?
```
[the remark this answers is not in the document]
Comment (Jolg42) [reply packages/client/src/__tests__/validatePrismaClientOptions.test.ts:53]: This change looks good, thanks!

**Review by Jolg42 — APPROVED** [review]
Thanks @jsoref for your first PR here!

It's nice and clean, also tests are passing so 🎊 

I added some small suggestions, after that let's merge your PR :shipit:

File: packages/client/scripts/postinstall.js (hunk 6 above)
[the remark this answers is not in the document]
Comment (Jolg42) [reply packages/client/scripts/postinstall.js:285]: For into/in it seems that it's better to rewrite the sentence here 🤔 
Suggestion is here https://github.com/prisma/prisma/pull/15645#discussion_r986545303

**Comment by jsoref** [comment]
@Jolg42 thanks. I like your changes a lot. Applied. 

I'm away until Thursday. See you on the other side.

**Comment by Jolg42** [comment]
The auto-merge didn't wait 😅 But it should be fine, tests were passing before.

Thanks for your contribution!

**Comment by jsoref** [comment]
Apparently the other checks weren't "required".

BTW, on the other side, it'd be nice if the ci didn't fail on forks (it requires self hosted runners that forks aren't likely to have). I've helped one project adjust a similar workflow to be friendly to forks.

**Comment by Jolg42** [comment]
@jsoref That's a good point, the self-hosted runners are quite recent and making that more friendly would be great! (thanks for the reminder)
Feel free to send a PR, I'll be happy to review 💚 
(Or if it's better that we do it, create an issue with what you recommend.)

**Comment by jsoref** [comment]
@Jolg42 gladly! #15653 -- thanks in advance and good luck. If you have questions, I might be able to answer on Thursday (or late next week).

**Pull request merged**